packages/core-data/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Enhancements
```

Comment:
This should be in `packages/DATAVIEWS/changelog`, not in core-data :) Also, it looks like your editor adds some formatting to other areas of the changelog.